src/extension/features/accounts/ctrl-enter-cleared/settings.js

```diff
@@ -0,0 +1,8 @@
+module.exports = {
+  name: 'CtrlEnterCleared',
+  type: 'checkbox',
+  default: true,
```

Comment:
This setting should be false as it changes a default behavior. It's pretty rare that we set a feature to true.